pkg/common/telemetry/lru.go

```diff
@@ -0,0 +1,46 @@
+package telemetry
```

Comment:
- probably the metrics belong in `telemetry/agent` somewhere
- why do we need locking?

Reply:
I thought we needed locking of the shared access to the same metrics object that holds the IncrCounter call since access to the shared metrics object happens in LRU cache's `UpdateEntries` and `UpdateSVIDs` functions.

Reply:
moved the lru.go metrics file to the agent package and removed the struct